source/release-notes.rst

```diff
@@ -21,6 +21,85 @@ Retail, Education, Workload consolidation and Marine segments.
    :local:
    :depth: 1
 
+CIV_02.22.04.50_A12
+===================
+
+Intended audience
+-----------------
+
+* Celadon Open Source Community who has subscribed to celadon@lists.linuxfoundation.org
+
+Customer support
+----------------
+
+* subscribe/unsubscribe celadon mailing list using : https://lists.linuxfoundation.org/mailman/listinfo/celadon
+
+Introduction
+------------
+Feature Details
+---------------
+* Celadon Upstreamed fixes in this release
+
+				* 99.97% Google* conformance on Alder Lake (ADL)platform with production ready
+				* Supporting Alder Lake (ADL) on single baseline and Single Image
+				* IoT configuration support SRIOV mode with 99.97% conformance
+				* IoT configuration support GVT-d with no conformance
```

Comment:
Remove below line, not applicable to ADL. Aligned with Basu for this.
* IoT configuration support GVT-d with no conformance